x-pack/plugins/ml/common/openapi/ml_apis_v2.json

```diff
@@ -0,0 +1,226 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "version" : "1",
+    "title" : "Machine learning APIs",
+    "description": "Kibana APIs for the machine learning feature",
+    "termsOfService" : "",
+    "license": {
+      "name": "Elastic License 2.0",
+      "url": "https://www.elastic.co/licensing/elastic-license"
+    }
+  },
+  "host" : "localhost:5601",
+  "basePath" : "/",
+  "tags" : [ {
+    "name" : "ml",
+    "description": "Machine learning"
+  } ],
+  "schemes" : [ "https" ],
+  "security" : [
+    {
+      "basicAuth": [ ]
+    },
+    {
+      "apiKey" : [ ]
+    }
+  ],
+  "paths" : {
+    "/s/{spaceId}/api/ml/saved_objects/sync" : {
```

Comment:
In the curl example generated in the preview, do you know why it uses `-u USER:KEY` rather than `-u USER:PASSWORD` ?

```
curl -XGET https://localhost:5601/s/{spaceId}/api/ml/saved_objects/sync \
-u USER:KEY
```

Reply:
That seems like a simple fix in the script, so I'll open a PR.